### DISCUSSION TITLE
Making repeaters in Campaign Monitor

### DISCUSSION THREAD

**Discussion by 901284** [mjmlio/mjml#2618] [Q&A]
Hi all,

Trying to make a repeater block in Campaign Monitor, but upon wrapping my mj-section in <repeater> I get the error 'Element repeater doesn't exist or is not registered'. If I wrap the whole block in <mj-raw> it works, but obviously it doesn't parse the MJML and just comes out as plain HTML.

Would really appreciate any thoughts or insights, cheers.

<img width="684" alt="Screen Shot 2023-01-12 at 4 35 15 pm" src="https://user-images.githubusercontent.com/74296302/212017596-8f8ad48f-3583-4d69-a8e1-025e287f3ecf.png">

**Comment by iRyusa** [comment]
You need to wrap mj-raw around your templating tags.

```
<mj-raw><repeater></mj-raw>... <mj-raw></repeater></mj-raw>
```

**Comment by addedlovely** [comment]
For anyone else googling and ending up here, if you want to apply the campaign monitor template tag `editable` to an `<img>`, simply give it the css-class="editable" and use the html-attributes head component to apply the tag.
https://documentation.mjml.io/#mj-html-attributes

` <mj-image src="assets/images/test.jpg" css-class="editable"></mj-image>
`

```
<mj-html-attributes>
    <mj-selector path=".editable img">
        <mj-html-attribute name="editable"></mj-html-attribute>
    </mj-selector>
</mj-html-attributes>
```
